bindings/ios/MEGARequest.h

```diff
@@ -484,6 +485,20 @@ typedef NS_ENUM (NSInteger, MEGANodeAccessLevel) {
  */
 @property (readonly, nonatomic) NSInteger numDetails;
 
+/**
+ * @brief Returns a dictionary of mega string list.
+ */
+@property (readonly, nonatomic) NSDictionary<NSString *, MEGAStringList*> *megaStringListDictionary;
+
+/**
+ * @brief Gets the string table response from a request.
+ *
+ * This value is valid for these requests:
+ * - [MEGASdk getRegisteredContacts:] - Obtains the user contacts registered in MEGA and verificated through SMS.
+ *
+ */
+@property (readonly, nonatomic) NSArray<MEGAStringList *> *megaStringTableArray;
```

Comment:
@jcmartinac , I think maybe `megaStringListArray` is a better name? the term `table` can be quite confusing in naming. It doesn't imply an Array.

Reply:
I use `table` following the naming of the SDK property. It think this is hard to name it in iOS but after talking with Navarro and Santana (and refactoring the code), `megaStringListArray` doesn't make sense because we are not going to use `MEGAStringList` in the return object. For the moment it is called `stringTableArray` without _mega_ prefix. But I think we will discuss this in the near future.

Reply:
Thanks Carlos. Yes, the `table` in SDK means a list of list or array of array. If you use `megaStringTableArray`, I would think this is a 3 layers list. 

I am OK to remove `mega` prefix. Maybe name it to `stringTable` if you want to follow its SDK naming.